src/components/outdated-version-banner.vue

```diff
@@ -0,0 +1,114 @@
+<!--
+Copyright 2024 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+
+<template>
+  <div v-if="showBanner" class="outdated-version-banner">
+    <iframe :src="iframeSrc" :title="$t('title')"></iframe>
+
+    <a class="btn btn-primary"
+      href="https://docs.getodk.org/central-upgrade/"
+      target="_blank"
+      v-tooltip.aria-describedby="$t('instructionsToUpgradeTooltip')">
+        {{ $t('instructionsToUpgrade') }}
+    </a>
+
+    <button class="btn btn-danger"
+      type="button"
+      v-tooltip.aria-describedby="$t('dismissTooltip')"
+      @click="dismiss">
+        {{ $t('dismiss') }}
+    </button>
+  </div>
+</template>
+
+<script setup>
+import { computed, inject } from 'vue';
+import { useRequestData } from '../request-data';
+import { useSessions } from '../util/session';
+
+const { visiblyLoggedIn } = useSessions();
+
+const { centralVersion, currentUser } = useRequestData();
+
+defineOptions({
+  name: 'OutdatedVersionBanner'
+});
+
+const container = inject('container');
+const { i18n: globalI18n } = container;
+const locale = computed(() => globalI18n.locale);
+
+const iframeSrc = computed(() => `https://getodk.github.io/central/outdated-version.html?version=${centralVersion.data?.currentVersion}&lang=${locale.value}`);
```

Comment:
If the iframe is rendered, I think that means that `centralVersion.dataExists`. So I'm thinking the `?.` could be removed.